frontend/app/src/store/defi/protocols/index.ts

```diff
@@ -645,8 +650,7 @@ export const useDefiSupportedProtocolsStore = defineStore(
             balances.push({
               address,
               protocol: DefiProtocol.MAKERDAO_DSR,
-              asset:
-                'eip155:1/erc20:0x6B175474E89094C44Da98b954EedeAC495271d0F',
+              asset: getIdentifierBySymbol('DAI'),
```

Comment:
Is there any reason here to do getIdentifier and expose it rather than importing the from data doing `.DAI`?